CSS_Tasks/CSS_Advanced/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="bg">
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <title>WebGL</title>
+    <link href="stylesheet.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <button type="submit" class="button greenButton"> Вход </button>
+        <button type="submit" class="button greenButton"> Регистрация </button>
+    </header>
+
+    <main class="page-container">
+        <div class="navigation-bar">
+
+            <nav class="navigationBar">
```

Comment:
You should follow one naming convention for classes. In order to escape duplicate names you can use just "navigation" here instead of "navigationBar".